Sources/S4.swift

```diff
@@ -0,0 +1,314 @@
+public typealias Byte = UInt8
```

Comment:
We should split these up into individual files. 

`HTTPRequest.swift`, `HTTPResponse.swift`, etc.